src/com/google/javascript/jscomp/AdvancedCoverageInstrumentationCallback.java

```diff
@@ -0,0 +1,205 @@
+package com.google.javascript.jscomp;
```

Comment:
All Java files need a license included. You can copy one from any other file in the repo like CoverageInstrumentationCallback.java.